packages/react-composites/tests/browser/call/hermetic/RoomsCallScreen.test.ts

```diff
@@ -71,3 +79,86 @@ test.describe('Rooms CallScreen tests for different roles', async () => {
     );
   });
 });
+
+test.describe('Rooms Participant RemoveButton tests for different roles', async () => {
+  test('Remove button is enabled for Presenter', async ({ page, serverUrl }, testInfo) => {
+    test.skip(isTestProfileStableFlavor());
+    const paul = defaultMockRemoteParticipant('Paul Bridges');
+    const participants = [paul];
+    const initialState = defaultMockCallAdapterState(participants);
+    await page.goto(buildUrlWithMockAdapter(serverUrl, initialState, { role: 'Presenter' }));
+    await waitForSelector(page, dataUiId(IDS.videoGallery));
+
+    if (!isTestProfileDesktop(testInfo)) {
+      await pageClick(page, dataUiId('call-with-chat-composite-more-button'));
+      const drawerPeopleMenuDiv = await page.$('div[role="menu"] >> text=People');
```

Comment:
Use `waitForSelector` here. It takes screenshot on timeout and also has in-built logic to improve local debugging runs.

Don't use playwright API for selectors directly.

Reply:
done